src/routes/MapExplorer/MapExplorer.store.js

```diff
@@ -184,11 +184,11 @@ const groupsQuery = `query (
 }`
 
 // actions
-export function fetchPostsForMap ({ activePostsOnly, context, slug, sortBy, search, filter, topics, boundingBox, groupSlugs, types }) {
+export function fetchPostsForMap ({ activePostsOnly, childPostInclusion = 'yes', context, slug, sortBy, search, filter, topics, boundingBox, groupSlugs, types }) {
```

Comment:
why 'yes' instead of just a boolean here?

Reply:
there was somewhere in the chain of passing the value around that was checking the 'truthiness' of the value, and that was causing problems.

I think it was this pattern

`  const defaultChildPostInclusion = get('settings.streamChildPosts', currentUser) || 'yes'
  const childPostInclusion = getQuerystringParam('c', state, props) || defaultChildPostInclusion`
  
  We determine the default for a bunch of settings by running it through a series of checks like this... but in this instance if it find a 'false' setting, it would ignore it

Reply:
ok, would be nice to comment on it at least